cumulusci/tasks/metadata/package.py

```diff
@@ -355,11 +353,22 @@ def check_delete_excludes(self, item):
             return True
 
 
-class AuraBundleParser(MetadataFilenameParser):
+class BundleParser(BaseMetadataParser):
     def _parse_item(self, item):
-        if item.startswith("."):
-            return []
-        return [item]
+        members = []
+        path = self.directory + "/" + item
+
+        # Skip non-directories
+        if not os.path.isdir(path):
+            return members
+
+        # Add the member if it is not namespaced
+        if "__" not in item:
```

Comment:
When would an Aura or LWC bundle contain a namespaced member? I don't think we need this condition.

Reply:
All Bundles should be in the local namespace for the repo.  I can't think of a way that another namespace's Bundle would be included in the metadata.

Reply:
Changed pushed